auslib/blobs/apprelease.py

```diff
@@ -700,7 +700,7 @@ def getHeaderXML(self, updateQuery, update_type, whitelistedDomains, specialForc
 
     def getInnerXML(self, updateQuery, update_type, whitelistedDomains, specialForceHosts):
         xml = []
-        xml.append('    <update type="%s" unsupported="true" detailsURL="%s" displayVersion="%s">' % (update_type, self['detailsUrl'], self["displayVersion"]))
+        xml.append('    <update type="%s" unsupported="true" detailsURL="%s" displayVersion="%s">' % (update_type, self['detailsUrl'].replace("%LOCALE%", updateQuery['locale']).replace("%VERSION%", updateQuery['version']), self["displayVersion"]))
```

Comment:
This line is quite long - can you split it up? It might be best to assign the substituted url to a temporary variable, and use that in this substitution.